packages/tasks/src/model-libraries.ts

```diff
@@ -616,6 +616,14 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		filter: false,
 		countDownloads: `path:"Open_Sora_v2.safetensors"`,
 	},
+	outetts: {
+		prettyLabel: "Outetts",
+		repoName: "Outetts",
```

Comment:
```suggestion
		prettyLabel: "OuteTTS",
		repoName: "OuteTTS",
```

(from what I can see in https://github.com/edwko/OuteTTS)